Add music#27

damwhit opened the pull request:
Danger zone now plays when game starts and stops when game ends.
cloud image size has been changed for more realistic collision

damwhit committed:
change cloud image size

damwhit committed:
danger zone plays during game

brianrip commented on lib/index.js:
```diff
@@ -5,11 +5,16 @@ $('.game-canvas').hide();
-  var game = new Game();
-  game.start();
+  var game2 = new Game(dangerZone);
+  game2.start();
```
Lets change game2 to nextGame so that it represents it dynamically.

damwhit committed:
change name to nextgame to for better representation

need to have nextgame variable to prevent game loop from recreating game ontop of self
